change/@microsoft-teams-js-7130594c-7f2f-47d9-80ff-229561e1845c.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "Added API version tags to barCode, calendar, call, clipboard, mail, marketplace and media capabilities",
```

Comment:
>barCode, calendar, call, clipboard, mail, marketplace and media [](http://example.com/codeflow?start=40&length=63)

Can you surround the capability names in backticks? Thanks!

Reply:
I will do it later, thanks for reminding. And will update this for rest of PRs.